### DISCUSSION TITLE
Order of protocols

### DISCUSSION THREAD

**Discussion by itagagaki** [1Remote/1Remote#816] [General]
RDP - RemoteApp ? - SSH - VNC - Telnet - FTP - Serial - SFTP - App ?

I have a feeling that this order is a bit odd, is there a reason for this?

![image](https://github.com/user-attachments/assets/a878fa7b-a627-403f-a275-ed074ccc6b8c)

**Comment by VShawn** [comment]
I arranged this order without any particular reason

**Reply by itagagaki** [reply]
How about this?

![2024-12-19 11 12 55](https://github.com/user-attachments/assets/648d72dd-f84d-4391-88a9-91616e9519bb)

**Reply by VShawn** [reply]
Uh Are you planning to reorder them or group them?

**Reply by itagagaki** [reply]
My suggestion is to simply change the order as above.

**Reply by VShawn** [reply]
I think we can follow your idea, tips: you just need to change the return value of `GetListOrder` of each protocols to change their ordering.

https://github.com/1Remote/1Remote/blob/main/Ui/Model/Protocol/RDP.cs#L713-L716


'GetListOrder' was intended to allow the user to customize the order, but it's overkill so the plan was abandoned

**Comment by itagagaki** [comment]
Changed (#817) and satisfied. Thank you very much!
